src/homework/day9/NumbersRunner.java

```diff
@@ -0,0 +1,22 @@
+//Создать поток данных целых чисел numbers из 8 чисел (3342, 34, 79, 23426, 68, 1324, 55, 7699) через (Stream.of())
+//В потоке данных проитерировать числа и отпечатать числа в консоль новой строки
+//Отсортировать и снова отпечатать в консоль с новой строки
+//Посчитать сумму всех чисел и вывести результат в консоль
+
+package homework.day9;
+
+import java.util.stream.Stream;
+
+public class NumbersRunner {
+    public static void main(String[] args) {
+        Stream<Integer> numbers = Stream.of(3342, 34, 79, 23426, 68, 1324, 55, 7699);
+        numbers.forEach(System.out::println);
+        int number = Stream.of(3342, 34, 79, 23426, 68, 1324, 55, 7699)
+                .sorted()
+                .peek(System.out::println)
+                .mapToInt(Integer::intValue).sum();
+        System.out.println("Сумма всех чисел:" + number);
```

Comment:
Instead of printing out a var from line 14 you can simply wrap lines 14-17 into sout() (without setting up the int number)